test/clj/game_test/cards/operations.clj

```diff
@@ -180,6 +180,20 @@
     (is (= 1 (:click (get-corp))))
     (is (= 7 (count (:hand (get-corp)))) "Drew 2 cards")))
 
+(deftest building-blocks
+  ;; Building Blocks - install and rez a barrier from HQ at no cost
+  (do-game
+    (new-game (default-corp ["Building Blocks" "Ice Wall"])
+              (default-runner))
+    (core/gain state :corp :credit 1)
+    (play-from-hand state :corp "Building Blocks")
+    (is (= 1 (:credit (get-corp))) "Corp starts with 1 credit")
```

Comment:
I'd like this to check that Building Blocks cost 5 to play. Same effect, but different error message.